recipe/meta.yaml

```
@@ -1,49 +1,61 @@
 {% set name = "tika" %}
-{% set version = "2.6.0" %}
-{% set compress_type = "tar.gz" %}
-{% set hash = "56670eb812944eb25ed73f1b3b075aa41e7a135b74b240822f28b819e5b373da" %}
+{% set version = "3.1.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.{{ compress_type }}
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
-  sha256: {{ hash }}
+  - url: https://github.com/chrismattmann/{{ name }}-python/archive/refs/tags/{{ version }}.tar.gz
+    sha256: ec234cbf5c192612cb50793ff2cd6b80c7be5f18688e6b19e4e1d2490e07630f
+    patches:
+      - patches/fix_test.patch
 
 build:
-  noarch: python
-  preserve_egg_dir: true
   entry_points:
     - tika-python = tika.tika:main
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vvv
 
 requirements:
   host:
-    - pip
     - python
```

https://github.com/chrismattmann/tika-python/blob/3.1.0/setup.py#L104
```suggestion
    - python
    - wheel
    - setuptools
```